### PR TITLE
WIP: Reset PIN Flow

### PR DESCRIPTION
**Pull request by rayandrie**

### CODE REVIEW

**Comment by rayandrie** [comment]
Still need to test

**Comment by rayandrie** [comment]
tested for email

File: Source/Cotter.swift [hunk 1]
```diff
@@ -137,13 +137,21 @@ public class Cotter {
...
     ) {
-        // hide the close button
+        // Hide the close button
         self.transactionPinVC.hideCloseButton = hideClose
         
         Config.instance.transactionCb = transformCb(parent: vc, cb: cb)
         
+        // Add user info if exist - name, sending method, etc.
+        if let name = name, let sendingMethod = sendingMethod, let sendingDestination = sendingDestination {
+            Config.instance.userInfo = UserInfo(name: name, sendingMethod: sendingMethod, sendingDestination: sendingDestination)
```
Comment (albertpurnama) [inline Source/Cotter.swift:155]: I think this should be set inside the configuration parameter of Cotter initialization?

**Review by albertpurnama — COMMENTED** [review]
We should definitely make unit tests for our SDK

File: Source/Cotter.swift (hunk 1 above)
Comment (rayandrie) [reply Source/Cotter.swift:155]: good call

**Review by albertpurnama — APPROVED** [review]

**Pull request merged**